ledger/ledger-configuration/src/main/scala/com/daml/ledger/configuration/Configuration.scala

```diff
@@ -21,13 +21,23 @@ final case class Configuration(
 )
 
 object Configuration {
+  val NoGeneration: Long = 0
+  val StartingGeneration: Long = 1
 
   /** Version history:
     * V1: initial version
     * V2: added maxDeduplicationTime
     */
   val protobufVersion: Long = 2L
 
+  val reasonableMaxDeduplicationTime: Duration = Duration.ofDays(1)
```

Comment:
Can we add a comment as to why is 1 day a reasonable deduplication time?

Reply:
The original value was chosen more or less randomly, because we thought one day is long enough for applications to recover and retry failed commands. AFAIK some ledgers will want to set this to one week though.

Participants may have to keep command deduplication data for up to this period of time. The only reason for having an upper bound on this duration is to avoid unbounded storage requirements.

Reply:
Done.

Reply:
GitHub is not refreshing comments for me until I hit Refresh. That's fun.

I'd be happy for driver authors to make this bigger; the important thing, as @rautenrieth-da says, is that it's bounded. However, I do think 1 day is totally reasonable for most set-ups.